drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTableModellerViewImpl.java

```diff
@@ -133,10 +132,18 @@ public void execute() {
 
     private final RestrictedMousePanMediator mousePanMediator = restrictedMousePanMediator();
 
+    @Inject
     private ColumnManagementView actionsPanel;
 
+    @Inject
     private ColumnManagementView conditionsPanel;
 
+    @Inject
+    private Instance<AttributeColumnConfigRow> attributeColumnConfigRows;
```

Comment:
Sorry.. I missed this last time around.. best to make this ```ManagedInstance``` to ensure instances are released.